ring/func/base.py

```diff
@@ -619,13 +623,13 @@ def __init__(self, *args, **kwargs):
                 super(RingRope, self).__init__(*args, **kwargs)
                 self.user_interface = self.user_interface_class(self)
                 self.storage = self.storage_class(self, storage_backend)
-
                 _ignorable_keys = suggest_ignorable_keys(
                     self.callable, ignorable_keys)
                 _key_prefix = suggest_key_prefix(self.callable, key_prefix)
+                inmemory_storage = True if hasattr(self.storage, 'inmemory_storage') else False
```

Comment:
`hasattr(...)` itself already is a `bool` expression. So ternary if-else operator is not required.